testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -3322,6 +3324,94 @@ public void testCreateSchemaWithNonLowercaseOwnerName()
         assertUpdate(newSession, "DROP SCHEMA " + schemaName);
     }
 
+    @Test
+    public void testCreateOrReplaceTableWhenTableDoesNotExists()
```

Comment:
`testCreateOrReplaceTableWhenTableDoesNotExists` -> `testCreateOrReplaceTableWhenTableDoesNotExist`

update also reference in the below comment (line 3352) to it